code/src/index.css

```diff
@@ -5,9 +5,195 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(238, 237, 237);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+
+
+.content-container {
+  max-width: 700px;
+  margin: auto;
+}
+/* Error board */
+.error-board {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.api-error {
+  text-align: center;
+  font-size: 30px;
+  color: palevioletred;
+  font-style: Trispace;
```

Comment:
Maybe the font-style could be set as global since you use it in several elements?👀